wifiphisher/pywifiphisher.py

```diff
@@ -489,21 +523,24 @@ def start(self):
         except (interfaces.InvalidInterfaceError,
                 interfaces.InterfaceCantBeFoundError,
                 interfaces.InterfaceManagedByNetworkManagerError) as err:
+            logger.exception("Setting {} to monitor mode".format(mon_iface))
```

Comment:
When the exception raises, I found that the `mon_iface` is not defined and will raise `variable not defined exception`.
The testing scenario: only one card that supports `monitor mode` on the system.

Reply:
@blackHatMonkey maybe we can fix it at #762 together.

Reply:
I think we should keep it separate as looking back at the logs it doesn't fit.